.github/workflows/frontend.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         run: ./mvnw -B install -DskipTests -DskipRat -pl ${INTERPRETERS} -Phadoop2 -Pscala-2.11
       - name: Run headless test
         run: xvfb-run --auto-servernum --server-args="-screen 0 1024x768x24" ./mvnw verify -DskipRat -pl zeppelin-web -Phadoop2 -Pscala-2.11 -Pweb-e2e -B
+      - name: Print zeppelin logs
+        if: always()
```

Comment:
Why is this needed ?

Reply:
Just for diagnose in case test fail, we can check the zeppelin server logs via this action